reference/enterprise-api-ref/build-api.markdown

```diff
@@ -0,0 +1,829 @@
+---
+layout: default
+title: Build API
+published: true
+tags: [reference, enterprise, API, build, modules]
+---
+
+The Build API enables you to easily manage policy projects and their respective CFEngine Build modules.
+
+# Projects API
+
+A project is a set of CFEngine Build modules and custom files/json/policy files.
+
+## Create project
+
+**URI:** https://hub.cfengine.com/api/build/projects
+
+**Method:** POST
+
+**Parameters:**
+
+* **repositoryUrl** *(string)*
+  Git repository URL. Project will be synchronized with this repository. Supported protocols: `http`, `https`, `ssh`
+  , `git`. Required. Git repository URL example: https://github.com/cfengine/build-index.git
```

Comment:
Rather than use build-index as an example of a project url we could throw up an exaple cfbs built repo and point to that, or just point to a completely fake one.